ext/web/10_filereader.js

```diff
@@ -368,6 +368,54 @@
       // alias for readAsArrayBuffer
       this.#readOperation(blob, { kind: "Text", encoding });
     }
+
+    get onerror() {
+      return FunctionPrototypeCall(makeEventHandlerGetter("error"), this);
```

Comment:
With this implementation a new getter/setter would be created for each `get` / `set`. Can we prevent that?

Reply:
haven't came up a solution which can prevent that problem, do you have any suggestion?

Reply:
Maybe create those up-front and assign to some private properties on the class?

Reply:
@bardiarastin  Sorry about that I cannot get what you mean. 

There is 6 pair of `get/set` for 6 event attribute. I thought it is too repeat/wordy too. We must explicitly defined those via class `get/set`, if we defined `get/set` via `Object.definedProperty` wpt test will fail cause is that the function name of the getters and setters is not correct.

BTW. I refactor `makeEventHandlerGetter` and `makeEventHandlerSetter` function to class private method, make event `get/set` function body more concise.

Reply:
If anybody have a better/DRY solution, feel free to add commit to this pr. I will very happy for your help.

Reply:
FYI. `jsdom` also implement webidl attribute using `Object.definedProperty` which may also encounter the same `get/set` function name issue with us. https://github.com/jsdom/jsdom/blob/ba8e5806d1a23960a03c3dd99af98fd0916b4bc9/lib/jsdom/living/helpers/create-event-accessor.js#L177